Marlin/SanityCheck.h

```diff
@@ -647,6 +647,13 @@
 #endif
 
 /**
+ * Nozzle park
+ */
+#if ENABLED(NOZZLE_PARK_FEATURE) && !HAS_BED_PROBE
+  #error Due to internal dependencies you must have a bed probe for NOZZLE_PARK_FEATURE to work
```

Comment:
This PR will be rebased Scott, now we have do_blocking_move_to_*() freeeeeeee ! ;-D